client/post-editor/media-modal/gallery/fields.jsx

```diff
@@ -144,13 +151,26 @@ export default React.createClass( {
 		);
 	},
 
+	renderReverseOrderButton() {
+		const { translate } = this.props;
+
+		return (
+			<EditorMediaModalFieldset>
+				<Button onClick={ this.updateReverseOrder } disabled={ this.props.settings.orderBy === 'rand' } >
+					{ translate( 'Reverse Order' ) }
+				</Button>
+			</EditorMediaModalFieldset>
+		);
+	},
+
 	render() {
 		const types = this.getTypeOptions();
 		const links = this.getLinkOptions();
 		const sizes = this.getSizeOptions();
 
 		return (
 			<div className="editor-media-modal-gallery__fields">
+				{ this.renderReverseOrderButton() }
```

Comment:
I've started encouraging more rendering inline in the `render` function when possible, as it keeps all render logic colocated. Here, since `renderReverseOrderButton` is simply returning another element, it would be straightforward to move that element in place of the function call here directly. Only in cases where there's conditional render logic does it become more complicated.